### DISCUSSION TITLE
Airflow Concurrency doesn't allow multiple rerunning tasks

### DISCUSSION THREAD

**Discussion by yonghyeokrhee** [apache/airflow#18823] [General]
Starting with Gannt, my concurrency for Spark SSH job is 15 and it doesn't have any problem at the beginning.
But at some points, failed job is accumulated and finally doesn't allow more than 2 jobs in the same time. 

![image](https://user-images.githubusercontent.com/30429223/136498463-1abf6073-e1a4-46e4-843e-d6196020da11.png)

Spark history says it has succeeded the job and so does Hadoop node manager.
It only the matter when in comes to Airflow.
I asked this to Stackoverflow but ain't got any answer.

My Airflow version is 1.10.12

many thanks.

**Comment by potiuk** [comment]
There is no way with that level of details to get any help - logs, stactraces, suspicious traces you looked at could be useful. But also don't expect too much of a help. 1.10.12 is pretty old. The whole 1.10 line reached end-of-life in June and it won't get any fixes, so you should rather consider migrating to Airlfow 2- which is way faster, more feature-full (and it will receive upgrades and fixes including critical security fixes).

**Reply by yonghyeokrhee** [reply]
thanks potiuk! this is second time I get help from your reply. I should really move to airflow 2.

**Reply by potiuk** [reply]
Yep. You should.
